Dnn.AdminExperience/ClientSide/Roles.Web/src/components/roles/RoleEditor/index.jsx

```diff
@@ -194,6 +194,11 @@ class RolesEditor extends Component {
     render() {
         let { state, props } = this;
         const columnOne = <div key="editor-container-columnOne" className="editor-container">
+            <div className="editor-row divider" style={{marginTop:"-3rem"}}>
+                <Label
+                    label={resx.get("RoleId") + ": " + state.roleDetails.id}
+                    style={{ fontWeight: "bolder" }} />
+            </div>
```

Comment:
Looks like this comes from your other PR

Reply:
Yes. Sorry.